fluidreview/api.py

```diff
@@ -139,3 +150,45 @@ def request(self, method, url_suffix, **kwargs):
             self.session = self.initialize_session()
             response = getattr(self.session, method)(urljoin(BASE_API_URL, url_suffix), **kwargs)
         return response
+
+
+def process_user(fluid_user):
+    """
+    Create/update User and Profile model objects based on FluidReview user info
+
+    Args:
+        fluid_user (ReturnDict): Data from a fluidreview.serializers.UserSerializer object
+    """
+    user = User.objects.filter(email=fluid_user['email']).first()
+    if not user:
+        user = User.objects.create(email=fluid_user['email'], username=fluid_user['email'])
+    profile, _ = Profile.objects.get_or_create(user=user)
+    if not profile.fluidreview_id:
+        profile.fluidreview_id = fluid_user['id']
+        profile.name = fluid_user['full_name']
+        profile.save()
+
+
+def parse_webhook(webhook):
+    """
+    Attempt to load a WebhookRequest body as JSON and assign its values to other attributes.
+
+    Args:
+        webhook (WebhookRequest): WebhookRequest instance
+
+    """
+    try:
+        body_json = json.loads(smart_text(webhook.body))
+        for attr in get_field_names(WebhookRequest):
```

Comment:
What if the incoming payload has `id` in it? IMO we should explicitly enumerate the fields we want to set here

Reply:
will do

Reply:
PS Actually`get_field_names` filters out autocreated fields like `id`, so the fields returned for `WebhookRequest` are: 
```
['created_on', 'updated_on', 'body', 'status', 'user_email', 'user_id', 'submission_id', 'award_id', 'award_name', 'award_cost', 'amount_to_pay']
```
But `created_on` and `updated_on` probably should not be modified.